scrapers/parse_scrape_output.py

```diff
@@ -242,6 +243,12 @@ def parse_date(d):
       except:
         warns.appent(f"ICU ({v}) not a number")
       continue
+    if k.startswith("Intubated"):
+      try:
+        intubated = int(v)
+      except:
```

Comment:
Please avoid `bare exceptions` as discussed in PEP8 and https://realpython.com/the-most-diabolical-python-antipattern
```suggestion
      except (TypeError, ValueError):
```

Reply:
@cclauss Could you take a look at https://github.com/openZH/covid_19/pull/333